app/views/_macros/components.nunjucks

```diff
@@ -228,7 +228,7 @@
 {% endmacro %}
 
 {% macro panel(content) %}
-  <article class="panel">
+  <article class="panel reading-width">
```

Comment:
I think it will depend on the use. When used within [hernia](https://beta.nhs.uk/conditions/hernia) is feels redistributable but in other cases it might not be. So maybe we just use a div for the macro?